geppetto.js/geppetto-client/src/components/ComponentFactory.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const ComponentFactory = {
```

Comment:
is everything in the ComponentFactory actually needed? I remember that only a bit of the functionality is really used by the PythonCapability, but I may be wrong

Reply:
the rest is used by jupyter_geppetto, we actually need everything in there.